src/solutions/ode_solutions.jl

```diff
@@ -180,6 +184,58 @@ function SymbolicIndexingInterface.is_parameter_timeseries(::Type{S}) where {
     Timeseries()
 end
 
+const SolutionWithSavedSubsystem = ODESolution{T1,
+    T2,
+    T3,
+    T4,
+    T5,
+    T6,
+    T7,
+    T8,
+    T9,
+    T10,
+    T11,
+    T12,
+    T13,
+    T14,
+    T15,
+    T16} where {
+    T1, T2, T3, T4, T5, T6, T7, T8, T9, T10, T11, T12, T13, T14, T15, T16 <: SavedSubsystem}
```

Comment:
this seems a bit fragile. Why not a trait function?

Reply:
Yeah I guess that is a better option